apps/client-ui/scripts/generate-amplify-config.sh

```
@@ -0,0 +1,371 @@
+
+# Function to list available templates
+list_templates() {
+    print_info "Available Amplify configuration templates:"
+    echo ""
+    
+    local templates_path="$PROJECT_DIR/$TEMPLATES_DIR"
+    
+    if [[ -d "$templates_path" ]]; then
+        for template in "$templates_path"/*.yml; do
+            if [[ -f "$template" ]]; then
+                local template_name=$(basename "$template" .yml)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)